samples/net/dsa/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
In commit `dsa: example: ...`, please change the commit subject to `samples: net: dsa: ....`

Reply:
Ok.